spec/features/cargos/user_can_create_new_cargo_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+feature 'user can create new cargo' do
+  let!(:truck) { create(:truck) }
+
+  background do
+    visit(cargos_path)
+    click_link('New cargo')
+  end
+
+  scenario 'user can fill in new cargo' do
+    fill_in('Description', with: 'description')
+    fill_in('Title', with: 'title')
+    fill_in('Reference number', with: 0)
+    fill_in('Value', with: 1)
+    select(text: truck.license_plate, from: 'cargo_truck_id')
+    choose('cargo[paid]', option: 'true')
+    click_button('Create Cargo')
+
+    expect(page).to have_content('Cargos Index')
+    expect(current_path).to eq(cargos_path)
+
+    within('tbody') do
+      within(all('tr')[0]) do
+        expect(page).to have_content('description')
+        expect(page).to have_content('title')
+        expect(page).to have_content(0)
+        expect(page).to have_content(1)
+        expect(page).to have_content('true')
+      end
+    end
+
+    newly_created_cargo = Cargo.order(:created_at).last
+    expect(newly_created_cargo.description).to eq('description')
+    expect(newly_created_cargo.title).to eq('title')
+    expect(newly_created_cargo.reference_number).to eq(0)
+    expect(newly_created_cargo.value).to eq(1)
+    expect(newly_created_cargo.truck).to eq(truck)
+    expect(newly_created_cargo.paid).to eq(true)
```

Comment:
Optionally for slightly more readability, Rspec will allow you to do this
```
expect(newly_created_cargo).to be_paid
```

since `paid` returns a boolean value, Rspec constructs the `be_paid` matcher for you that you can call on `newly_created_cargo`